--- control	2025-06-29 19:18:58.934365397 +0200
+++ control	2025-07-01 18:16:31.777992888 +0200
@@ -8,7 +8,7 @@ Build-Depends:
  qt6-base-dev (>= 6.2.0~),
  qt6-tools-dev (>= 6.2.0~),
  qt6-tools-dev-tools (>= 6.2.0~),
- qt6-svg-dev (>= 6.2.0~),
+ libqt6svg6-dev (>= 6.2.0~),
 Standards-Version: 4.5.0
 Homepage: https://github.com/flameshot-org/flameshot
 Vcs-Browser: https://github.com/flameshot-org/flameshot
