From: juanma1980 <juanma1980@gmail.com>
Date: Thu, 21 Sep 2017 14:04:48 +0200
Subject: build patch

---
 flameshot.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flameshot.pro b/flameshot.pro
index d62071e..219ca49 100644
--- a/flameshot.pro
+++ b/flameshot.pro
@@ -15,7 +15,7 @@ greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 CONFIG    += c++11
 CONFIG    += link_pkgconfig
 
-#CONFIG    += packaging   # Enables "make install" for packaging paths
+CONFIG    += packaging   # Enables "make install" for packaging paths
 
 TARGET = flameshot
 TEMPLATE = app
